c2corg_api/models/user.py

```diff
@@ -0,0 +1,57 @@
+import bcrypt
+
+from sqlalchemy import (
+    Boolean,
+    Column,
+    Integer,
+    String
+    )
+
+from c2corg_api.models import Base, users_schema
+
+
+class User(Base):
+    """
+    Class containing the users' private and authentication data.
+    """
+    __tablename__ = 'user'
+    __table_args__ = {"schema": users_schema}
+
+    id = Column(Integer, primary_key=True)
+    username = Column(String(200), nullable=False, unique=True)
+    email = Column(String(200), nullable=False, unique=True)
+    email_validated = Column(Boolean, nullable=False)
+    _password = Column("password", String(255), nullable=False)
```

Comment:
Because it is the encrypted version of the password. It is set using ``_set_password``:
https://github.com/c2corg/v6_api/blob/oauth/c2corg_api/models/user.py#L30-L31